dipy/io/image.py

```diff
@@ -27,6 +27,16 @@ def save_nifti(fname, data, affine, hdr=None):
     result_img.to_filename(fname)
 
 
+def load_affine_matrix(fname):
```

Comment:
This is just a wrapper around `np.loadtxt`. Let's use that one.

Reply:
thank you! addressed in d90ea0d